src/server/pingserver-rs/Cargo.toml

```diff
@@ -8,7 +8,12 @@ version = "0.0.0"
 clap = "2.33"
 log = "0.4.8"
 libc = "0.2"
-pelikan = { path="../../rust-util/pelikan", features=["protocol_admin"] }
+pelikan = { path="../../rust-util/pelikan", features=["protocol_admin", "protocol_ping"] }
+rustcore = { path="../../rustcore" }
+
+lazy_static = "1.4"
+logging-allocator = "0.1"
+backtrace = "*"
```

Comment:
Please use a specific version of the dependency. Ordering alphabetically would be nice too.

Reply:
This dependency isn't even used within the crate anymore - I've removed both it and logging-allocator.